build.rs

```diff
@@ -0,0 +1,50 @@
+#[cfg(feature = "inline_asm")]
+fn main() {}
+
+#[cfg(all(not(feature = "inline_asm"), not(feature = "stable")))]
+fn check_inline_asm() {
+    compile_error!("Neither feature \"stable\" nor \"inline_asm\" was set!");
+}
+
+#[cfg(all(not(feature = "const_fn"), not(feature = "stable")))]
+fn check_const_fn() {
+    compile_error!("Neither feature \"stable\" nor \"const_fn\" was set!");
+}
+
+#[cfg(all(not(feature = "abi_x86_interrupt"), not(feature = "stable")))]
+fn check_abi_x86_interrupt() {
+    compile_error!("Neither feature \"stable\" nor \"abi_x86_interrupt\" was set!");
+}
```

Comment:
Same here: There is no stable replacement for the calling convention, so forcing to enable the stable feature does not make sense to me.

Reply:
removed